packages/web-components/src/tabs/tabs.stories.ts

```diff
@@ -0,0 +1,235 @@
+
+const tabsDisabledTabs = html`
+  <fluent-tabs disabled>
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+  </fluent-tabs>
+
+  <fluent-tabs>
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab disabled> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+  </fluent-tabs>
+`;
+export const TabsDisabled = renderComponent(tabsDisabledTabs).bind({});
+
+const tabsSizeSmall = html`
+  <fluent-tabs size="small">
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+  </fluent-tabs>
+  <fluent-tabs size="small" orientation="vertical">
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+  </fluent-tabs>
+`;
+export const TabsSizeSmall = renderComponent(tabsSizeSmall).bind({});
+
+const tabsSizeMedium = html`
+  <fluent-tabs size="medium">
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+  </fluent-tabs>
+  <fluent-tabs size="medium" orientation="vertical">
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+    <fluent-tab-panel></fluent-tab-panel>
+  </fluent-tabs>
+`;
+export const TabsSizeMedium = renderComponent(tabsSizeMedium).bind({});
+
+const tabsSizeLarge = html`
+  <fluent-tabs size="large">
+    <fluent-tab> First Tab </fluent-tab>
+    <fluent-tab> Second Tab</fluent-tab>
+    <fluent-tab> Third Tab</fluent-tab>
+    <fluent-tab> Fourth Tab</fluent-tab>
+
+    <fluent-tab-panel></fluent-tab-panel>
```

Comment:
For testing should we add some content in the tab panels?

Reply:
The reason i did not add content in many cases in the tab panels is because the extra space and content seemed to clutter the page and detract from illustrating the different examples of the tabs layout (vertical, subtle, small, medium, large, ect).